### PR TITLE
159-161: bugfix - filters on social page

### PR DESCRIPTION
**Pull request by KaterynaHerasymiva**

### CODE REVIEW

**Review by Anatolii-Ivanov — DISMISSED** [review]

File: frontend/src/app/shared/components/dropdown/dropdown.component.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
             </div>
         </div>
-        <div *ngIf="!selectedItems.length" class="dropdown-btn-title" >
+        <div *ngIf="selectedItems.length !== 1" class="dropdown-btn-title">
             {{getFullTitle()}}
         </div>
-        <div *ngIf="selectedItems.length" class="dropdown-btn-title-when-selected" >
+        <div *ngIf="selectedItems.length === 1" class="dropdown-btn-title-when-selected">
             {{getFullTitle()}}
         </div>
```
Comment (bochka123) [inline frontend/src/app/shared/components/dropdown/dropdown.component.html:13]: We have 2 similar div with only difference in class name, may be it`s better to have 1, but add ngClass or className

**Review by bochka123 — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/dropdown/dropdown.component.html (hunk 1 above)
Comment (tatianahutii) [inline frontend/src/app/shared/components/dropdown/dropdown.component.html:13]: use ngClass for this

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/dropdown/dropdown.component.html (hunk 1 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/dropdown/dropdown.component.html]: fixed

Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/dropdown/dropdown.component.html:13]: fixed

**Review by bochka123 — APPROVED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**